recipes/r-distillr/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "distillr" %}
+{% set version = '0.0.0.9000' %}
+{% set github = "https://github.com/alberdilab/distillr" %}
+{% set sha256 = '0b22d09dde91ee01f735074a8bfb3a7129a799fabf3aa21dcb77d5842f119be4' %}
+
+package:
+  name: r-{{ name }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    #- {{ github }}/archive/refs/tags/v{{ version }}.tar.gz
+     - https://github.com/cmkobel/distillR/archive/refs/heads/package.zip
```

Comment:
Please use a proper tag/release

Reply:
Yes of course. I'm just debugging a few issues before setting up the correct release channel.